frontend/mirage/config.js

```diff
@@ -77,28 +93,86 @@ export default function() {
 
   this.get(
     '/proxy/sysupport/subscription-projects',
-    'timed-subscription-project'
+    ({ timedSubscriptionProjects }, request) => {
+      if (request.queryParams.include) {
+        let includes = request.queryParams.include.split(',')
+        let index = includes.indexOf('billing_type')
+        if (index > -1) {
+          includes.splice(index, 1, 'billing-type')
+          request.queryParams.include = includes.join()
+        }
+      }
```

Comment:
Why is this necessary?

Reply:
The timed backend only knows the include field `billing_type`. Mirage fails since it wants to resolve the included models and there is no frontend model called `billing_type`. So either we solve this here or edit the serializer so we can use `billig-type` in the model hook.